content/_index.fr.md

```diff
@@ -16,6 +16,10 @@ hero:
       author: Programme Joinup de la Commission européenne
       title: Désigné « Logiciel open-source le plus innovant » en 2019
       link: https://joinup.ec.europa.eu/collection/sharing-and-reuse-it-solutions/sharing-reuse-awards-2019-results#oss-inno
+    - image: dpg.png
+      author: Digital Public Goods Alliance
```

Comment:
DPG Alliance didn't translate its name to French. So we keep it in English here but we try to translate the description.